ubuntu_cali_path: '/home/szhang/catkin_ws/src/t42_control/rollout_t42/cali_info/_nf_na_ws_wm_v0.1_0.cali'
mac_cali_path: '/Users/zsbjltwjj/Downloads/t42_hand/t42_control/rollout_t42/cali_info/_nf_na_ws_wm_v0.1_0.cali'